src/dictionaries/it.json

```diff
@@ -54,13 +54,13 @@
       "security": "Sicurezza",
       "selectauthmethod": "Seleziona la modalità di autenticazione che preferisci e accedi al tuo profilo IO in versione web.",
       "tooltipaccessabled": "L’accesso a IO è abilitato con tutte le tue identità digitali",
-      "unlockioaccess": "Per sbloccare l’accesso in app IO segui questi passaggi:\nVai sul sito io.italia.it\nAccedi con le tue credenziali SPID o CIE\nVai alla sezione Ripristina accesso a IO\nSegui i passaggi e quando richiesto inserisci il codice di ripristino",
-      "whatdo": "Cosa devi fare?",
+      "unlockioaccess": "Per sbloccare l’accesso in app IO segui questi passaggi:Vai sul sito <u>io.italia.it</u><ul><li>Accedi con le tue credenziali SPID o CIE</li><li>Vai alla sezione <em>Ripristina accesso a IO</em></li><li>Segui i passaggi e quando richiesto inserisci il <strong>codice di ripristino</strong></li></ul>",
```

Comment:
there is an error into the format! You have not inserted the first point "Vai sul sito <u>io.italia.it</u>" into the unorder list tag!